python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -55,6 +68,50 @@ def __init__(self, database, driver='sqlite', host=None, port=None):
         self.host = host
         self.port = port
         self.database = database
+        self._seeing_column = 'FWHMeff'
+
+        # a dict keyed on the user interface (i.e. args to getObservationMetaData)
+        # names of OpSim data columns.  Returns a tuple that is the
+        # (name of data in OpSim, transformation to go from user interface to OpSim units, dtyp in OpSim)
+        self._user_interface_to_opsim = {'obsHistID': ('obsHistID', None, np.int64),
```

Comment:
I don't want users to be able to change this, since it defines the expected OpSim schema and I think it should be our responsibility to keep that up-to-date.

Users should only care about this if they are trying to draw up their own mock OpSim cadence, in which case I think they should consult the OpSim documentation to learn what columns are contained in the Summary table.